community/modules/scheduler/schedmd-slurm-gcp-v5-controller/main.tf

```diff
@@ -93,39 +94,35 @@ module "slurm_controller_instance" {
 module "slurm_controller_template" {
   source = "github.com/SchedMD/slurm-gcp.git//terraform/slurm_cluster/modules/slurm_instance_template?ref=5.9.0"
 
```

Comment:
This feels unnecessarily painful to review.
I see now that: 
additional_networks was removed
nic_type was removed
total_egress_bandwidth_tier was removed

Reply:
Moving access_config to line 60 from 96 attaches the public IP address to the instance rathe than the template.

For the other settings, we haven't had gVNIC/bandwidth tier/multiple NIC functionality for controller or login nodes at any point. Adding them requires further code modifications to the SchedMD repository. The settings should be removed until a safe implementation can be developed or added in the v6 series.

Longer: 

Those settings were introduced in #1787 and adapted into #1794 (which was merged). Looking closely at this problem, they are broken in the same way as public IPs on the controller: setting them on the template will not do any good because they are overridden in the slurm_controller_instance and slurm_login_instance modules. There is no way to integrate the functionality into slurm_controller_instance and slurm_login_instance without code changes in the SchedMD repo.

Reply:
sorry, I did not notice that the move of access_config moved it to another resource, artifact of github view and lack of my attention to details. Thanks!